downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -30,6 +30,186 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
 
 ### New features
 
+#### Introduction of several features to workers
+
+Several new features have been introduced to enhance the capabilities of workers.  
+
+- Introduction of alternate receive action
+
+The alternate receive action can be used to receive values from multiple send actions. It operates by waiting until it encounters a non-error message, a panic termination status on a closed channel, or the closure of all channels. Alternate receive action sets the first non-error value it encounters as the outcome.
+
+```ballerina
+import ballerina/io;
+import ballerina/lang.runtime;
+
+public function main() {
+    worker w1 {
+        2 -> w3;
+    }
+
+    worker w2 {
+        runtime:sleep(2);
+        3 -> w3;
+    }
+
+    worker w3 returns int {
+        // The value of the variable `result` is set as soon as the values from either
+        // worker `w1` or `w2` is received.
+        int result = <- w1 | w2;
+        return result;
+    }
+
+    worker w4 returns error? {
+        int value = 10;
+        if value == 10 {
+            return error("Error in worker 4");
+        }
+        value -> w6;
+    }
+
+    worker w5 {
+        runtime:sleep(2);
+        3 -> w6;
+    }
+
+    worker w6 returns int|error? {
+        // Alternate receive action waits until a message that is not an error is received. 
+        // Since `w4` returns an error, it waits further and sets the value that is received from `w5`.
+        int a = check <- w4 | w5;
+        return a;
+    }
+
+    int w3Result = wait w3;
+    io:println(w3Result);
+
+    int|error? w6Result = wait w6;
+    io:println(w6Result);
+}
+```
+
+- Introduction of multiple receive action
+
+The multiple receive action can be used to receive values from multiple send actions. It operates by awaiting the receipt of values from all the send actions, subsequently constructing a map containing those values.
+
+```ballerina
+import ballerina/io;
+import ballerina/lang.runtime;
+
+type Result record {
+    int a;
+    int b;
+};
+
+public function main() {
+    worker w1 {
+        2 -> w3;
+    }
+
+    worker w2 {
+        runtime:sleep(2);
+        3 -> w3;
+    }
+
+    worker w3 returns Result {
+        // The worker waits until both values are received.
+        Result result = <- {a: w1, b: w2};
+        return result;
+    }
+
+    Result result = wait w3;
+    io:println(result);
```

Comment:
Do we need another worker here? Why not use the multiple receive action in the function worker?

Reply:
Thanks. This simplified all samples.